include/IrrCompileConfig.h

```diff
@@ -225,6 +233,10 @@ the engine will no longer read .png images. */
 #ifdef NO_IRR_COMPILE_WITH_TGA_LOADER_
 #undef _IRR_COMPILE_WITH_TGA_LOADER_
 #endif
+#define _IRR_COMPILE_WITH_KTX_LOADER_
+#ifdef NO_IRR_COMPILE_WITH_KTX_LOADER_
+#undef _IRR_COMPILE_WITH_KTX_LOADER_
+#endif
```

Comment:
nope make a CMAKE option and in the configure file, just like `IRR_BUILD_MITSUBA`